liferay-workflow/liferay-portal
# LPS-201366 Move outside, they should be let

Motivation
=======
We are removing AUI code from `portal-reports-engine-console-web`. 
Here's the ticket: https://liferay.atlassian.net/browse/LPS-201366

Steps to Verify
========
1. Go to Configuration -> Reports Admin -> Definition tab -> click on (+) button to add a Definition report
2. For the template file, create a `.jrxml` local file and copy the content from [this link](https://help.liferay.com/hc/en-us/articles/360018185011-Generating-Reports-in-Liferay).
3. In the definition view, in the Reports Parameters section, add different params and check that they are being added by saving the report and opening it again.
4. After saving the Definition, go back to its view and click on Add Schedule on the bottom of the view.
5. In the new view search for the Reports Parameter section and check that was working as before.
